kaedama.py

```diff
@@ -250,18 +259,34 @@ def main():
                 print(params[key])
                 pprint.pprint(locals())
                 params[key]=params[key].format(**locals())
+
+    if args.test_mode:
+        print("[TESTMODE: print parameter block]")
+        pprint.pprint(params)
                 
-                
 
-    filesystem    = config.get('filesystem',None)                         
+    # Default filesystem.  Override with vaules specified in the workflow.
```

Comment:
Filesystem entries may be specified for each directory in the yaml file.   If omitted, default will be used.